### PR TITLE
Prepare for v1.0.0-rc3 release

### PR DESCRIPTION
**Pull request by tgraf**
Signed-off-by: Thomas Graf <thomas@cilium.io>

### CODE REVIEW

**Comment by tgraf** [comment]
test-me-please

File: NEWS.rst [hunk 1]
```diff
@@ -2,6 +2,56 @@
 NEWS
 ******
 
+Version 1.0-rc4
+===============
+
+:date: 2018-01-18
+:commit: nil
+
+Changes:
```
Comment (aanm) [inline NEWS.rst]: add a newline, otherwise the bullet points don't work

File: NEWS.rst [hunk 2]
```diff
@@ -2,6 +2,56 @@
...
+* Allow for empty endpoint selector. This enables defining policy which applies to all endpoints. (#2358, @tgraf)
+* docs: Cluster-wide debugging tool documentation (#2356, @ashwinp)
+* Add CRD validation for CNP in kubernetes (#2304, @aanm)
+* Use DNS names in getting started guides (#2254, @techcet)
+* use cilium/connectivity-container in nightly tests (#2247, @ianvernon)
+* fail all stages in build if any stage fails in Jenkins (#2246, @ianvernon)
+* Enabled policy enforcement on cilium network policy from any namespace (#2235, @aanm)
+
+Bugfixes Changes:
+* agent: Increase timeout when executing commands (#2512, @tgraf)
```
Comment (aanm) [inline NEWS.rst]: Here as well

**Review by aanm — CHANGES_REQUESTED** [review]

File: NEWS.rst (hunk 1 above)
Comment (tgraf) [reply NEWS.rst]: fixed

File: NEWS.rst (hunk 2 above)
Comment (tgraf) [reply NEWS.rst]: fixed

**Comment by tgraf** [comment]
test-me-please

File: NEWS.rst [hunk 3]
```diff
@@ -2,6 +2,60 @@
...
+Version 1.0-rc4
+===============
+
+:date: 2018-01-18
+:commit: nil
+
+Changes
+-------
+
+* Multi stage Docker builds to use prebuilt Envoy dependencies. (#2452, @jrajahalme)
```
Comment (aanm) [inline NEWS.rst]: Sorry I also noticed the link to the PR is not working since rc2. Can you replace all `#ID` with `ID_`?

**Review by aanm — CHANGES_REQUESTED** [review]

File: NEWS.rst (hunk 3 above)
Comment (tgraf) [reply NEWS.rst]: fixed. This is still a manual sed for now and not automated

**Comment by tgraf** [comment]
test-me-please

**Review by aanm — APPROVED** [review]

**Pull request merged**